src/signer_node.rs

```diff
@@ -197,6 +203,18 @@ impl<T: TapyrusApi, C: ConnectionManager> SignerNode<T, C> {
                 },
                 None => {}
             }
+
+            // Unix Signal handler
+            match unix_stop_signal_handler.load(Ordering::Relaxed) {
+                0 => {
+                    log::trace!("No stop signale");
```

Comment:
I think this `No stop` log is not necessary, it will be garbage log.

Reply:
I think it is ok because of  TRACE level log.